src/components/widgets/InputPopup.ts

```diff
@@ -194,11 +194,6 @@ export class InputPopup extends EventEmitter {
             return
         } // Continue only if the result is 0
```

Comment:
What does the mouseframe check do?

Reply:
the mouseframe check is necessary to distinguish whether or not the received input character is part of the mouse signal propagation protocol. In practice they are seen as special keys on the keyboard.
Without that check, these characters would cause malfunctions on all

Reply:
Ah ok that makes sense